pom.xml

```diff
@@ -56,6 +56,8 @@
     <json.version>20180813</json.version>
     <log4j.version>2.13.0</log4j.version>
     <axis2.version>1.7.9</axis2.version>
+    <javadoc.excludePackageNames></javadoc.excludePackageNames>
```

Comment:
there it isn't, It's just to ensure that it's cleared out for most projects. I set it in wsclient because there are certain packages we want to exclude from javadoc (generated stuff and antlr)